src/H5FDmpio.c

```diff
@@ -1349,6 +1343,12 @@ H5FD__mpio_read(H5FD_t *_file, H5FD_mem_t H5_ATTR_UNUSED type, hid_t H5_ATTR_UNU
     if (bytes_read < 0 || bytes_read > io_size)
         HGOTO_ERROR(H5E_IO, H5E_READERROR, FAIL, "file read failed")
 
+#ifdef H5FDmpio_DEBUG
+    if (H5FD_mpio_debug_r_flag)
+        HDfprintf(stderr, "%s: (%d) mpi_off = %ld  size_i = %lld\n", FUNC, file->mpi_rank, (long)mpi_off,
```

Comment:
Could you switch the printf string to say 'bytes_read' instead of 'size_t' now?

Reply:
That would make my post-processing easier. Hopefully, not too many users depend on it being "size_i" for their scripts. Switched.